docs/source/conf.py

```diff
@@ -56,9 +56,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.2.1'
+version = '2.0'
```

Comment:
```suggestion
version = '2.0.0'
```

Reply:
@anapaulagomes @berinhard but according to comment that stands for short version:
```
# The short X.Y version.
```